website/docs/policies/working_with_policies/unique_email.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Ensure unique email addresses
+---
+
+Due to the database design of authentik, Email addresses are by default not required to be unique. This behavior can however be changed by policies.
+
+The policy snippet below can be used both with enrollment flows, where the policy should be bound to any stage before the [User write](../../flow/stages/user_write.md) stage, or it can be used with the [Prompt stage](../../flow/stages/prompt/index.md).
```

Comment:
Can we say where the user needs to add this snippet? Is it added in the Expression field of the policy? Or... ?